backend/src/test/java/reviewme/review/repository/ReviewRepositoryTest.java

```diff
@@ -56,137 +57,194 @@ class ReviewRepositoryTest {
     @Nested
     @DisplayName("리뷰 그룹 아이디에 해당하는 리뷰를 생성일 기준 내림차순으로 페이징하여 불러온다")
     class FindByReviewGroupIdWithLimit {
-        Question question = 서술형_필수_질문();
-        Section section = 항상_보이는_섹션(List.of(question));
-        Template template = templateRepository.save(new Template(List.of(section)));
-        private final ReviewGroup reviewGroup = reviewGroupRepository.save(비회원_리뷰_그룹());
-
-        private final Review review1 = reviewRepository.save(
-                비회원_작성_리뷰(template.getId(), reviewGroup.getId(), null));
-        private final Review review2 = reviewRepository.save(
-                비회원_작성_리뷰(template.getId(), reviewGroup.getId(), null));
-        private final Review review3 = reviewRepository.save(
-                비회원_작성_리뷰(template.getId(), reviewGroup.getId(), null));
-
         @Test
-        void 페이징_크기보다_적은_수의_리뷰가_등록되었으면_그_크기만큼의_리뷰만_반환한다() {
+        void lastReviewId가_주어지지_않을_경우_가장_최근의_리뷰부터_반환한다() {
             // given
-            int limit = 5;
-            long lastReviewId = Long.MAX_VALUE;
+            ReviewGroup reviewGroup = reviewGroupRepository.save(회원_리뷰_그룹());
+            Review review1 = reviewRepository.save(비회원_작성_리뷰(1L, reviewGroup.getId(), List.of()));
+            Review review2 = reviewRepository.save(비회원_작성_리뷰(1L, reviewGroup.getId(), List.of()));
+            Review review3 = reviewRepository.save(비회원_작성_리뷰(1L, reviewGroup.getId(), List.of()));
+
+            Long lastReviewId = null;
 
             // when
-            List<Review> actual = reviewRepository.findByReviewGroupIdWithLimit(
-                    reviewGroup.getId(), lastReviewId, limit);
+            List<Review> reviews = reviewRepository.findByReviewGroupIdWithLimit(reviewGroup.getId(), lastReviewId, 10);
 
             // then
-            assertThat(actual)
-                    .hasSize(3)
-                    .containsExactly(review3, review2, review1);
+            assertThat(reviews).extracting(Review::getId)
+                    .containsExactly(review3.getId(), review2.getId(), review1.getId());
         }
 
         @Test
-        void 페이징_크기보다_큰_수의_리뷰가_등록되었으면_페이징_크기만큼의_리뷰를_반환한다() {
+        void lastReviewId가_주어질_경우_해당ID_이후의_가장_최근의_리뷰부터_반환한다() {
```

Comment:
```suggestion
        void lastReviewId가_주어질_경우_해당ID_이전의_가장_최근의_리뷰부터_반환한다() {
```

"이후"가 아니라 "이전"으로 변경되는게 좋겠네요!